test/Listener/_files/good/merge3.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-modulemanager for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-modulemanager/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-modulemanager/blob/master/LICENSE.md New BSD License
```

Comment:
invalid repo name

Reply:
I copy it from `/good/merge2.php`, this repo is https://github.com/laminas/laminas-modulemanager which already correct.